src/lib/coda_lib/coda_subscriptions.ml

```diff
@@ -97,13 +102,8 @@ let create ~logger ~wallets ~time_controller ~external_transition_database
               let best_tip_diff_pipe =
                 Transition_frontier.best_tip_diff_pipe transition_frontier
               in
-              Broadcast_pipe.Reader.iter best_tip_diff_pipe
-                ~f:(fun {removed_user_commands; _} ->
-                  ( match removed_user_commands with
-                  | [] ->
-                      ()
-                  | _ ->
-                      Strict_pipe.Writer.write writer () ) ;
+              Broadcast_pipe.Reader.iter best_tip_diff_pipe ~f:(fun _ ->
```

Comment:
I suggested this but I just noticed a problem if the `iter` starts after the `New_frontier` diff has already been written. It will erroneously skip the first diff assuming it was generated from `New_frontier`, but it isn't.